Data/Shaders/NormalVertexShader.glsl

```diff
@@ -0,0 +1,35 @@
+// VERTEX SHADER
+#version 330 core
+
+layout(location = 0) in vec3 inVertexPositionModelSpace;
+layout(location = 1) in vec2 inVertexUVs;
+layout(location = 2) in vec3 inNormal;
+layout(location = 3) in vec3 inTangent;
+
+out LightData {
+	vec3 objectColor;
```

Comment:
This is unrelated to light data since it reflects the color of objects in the scene and not a light and therefore should probably not be in LightData

Reply:
Also consider if you actually need this attribute I haven't seen this used before (only sparsely seen vertex color used)